data/layout/telescope_positions-North-4MST.ecsv

```diff
@@ -1,58 +1,51 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
-# - {name: telescope_name, datatype: string}
-# - {name: pos_x, unit: m, datatype: float64}
-# - {name: pos_y, unit: m, datatype: float64}
-# - {name: pos_z, unit: m, datatype: float64}
-# - {name: lat, unit: deg, datatype: float64}
-# - {name: lon, unit: deg, datatype: float64}
-# - {name: utm_east, unit: m, datatype: float64}
-# - {name: utm_north, unit: m, datatype: float64}
-# - {name: alt, unit: m, datatype: float64}
+# - {name: telescope_name, datatype: string, description: common telescope name}
+# - {name: pos_x, unit: m, datatype: float32, description: telescope position towards North relative to MST15 (central MST)}
+# - {name: pos_y, unit: m, datatype: float32, description: telescope position towards West relative to MST15 (central MST)}
+# - {name: pos_z, unit: m, datatype: float32, description: telescope position upwards relative to NNN}
 # meta:
+#   data_type: positionfile
+#   description: "telescope positions for CTA North (La Palma)"
 #   EPSG: 32628
 #   center_alt: !astropy.units.Quantity
 #     unit: &id001 !astropy.units.Unit {unit: m}
-#     value: 2150.0
+#     value: 2177
 #   center_easting: !astropy.units.Quantity
 #     unit: *id001
-#     value: 217654.06411838497
+#     value: 217611.227
 #   center_lat: !astropy.units.Quantity
 #     unit: &id002 !astropy.units.Unit {unit: deg}
-#     value: 28.762158
+#     value: 28.7621661
 #   center_lon: !astropy.units.Quantity
 #     unit: *id002
-#     value: -17.891571
+#     value: -17.8920302
 #   center_northing: !astropy.units.Quantity
 #     unit: *id001
-#     value: 3185065.2899643113
+#     value: 3185066.278
 #   corsika_obs_level: !astropy.units.Quantity
 #     unit: *id001
-#     value: 2158.0
+#     value: 2158
 #   corsika_sphere_center:
 #     LST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 16.0
 #     MST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 9.0
-#     SST: !astropy.units.Quantity
-#       unit: *id001
-#       value: 3.25
 #   corsika_sphere_radius:
 #     LST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 12.5
 #     MST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 9.6
-#     SST: !astropy.units.Quantity
-#       unit: *id001
-#       value: 3.0
+#   comments: |
+#       UTM Zone 28
 # schema: astropy-2.0
-telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
-MST-01 70.0 70.0 70.0 28.762789585465427 -17.892287768525946 217585.7457324182 3185137.0120037775 2149.0
-MST-02 -70.0 -70.0 -70.0 28.7615264106737 -17.892287759896902 217582.3421113845 3184996.971542539 2149.0
-MST-03 70.0 70.0 70.0 28.762789585465427 -17.890854231474055 217725.7861220474 3185133.6083178106 2149.0
-MST-04 -70.0 -70.0 -70.0 28.7615264106737 -17.8908542401031 217722.38250768976 3184993.567993115 2149.0
+telescope_name      pos_x      pos_y      pos_z
```

Comment:
Same here - would it make sense to use 4 realistic MST positions/

Reply:
Changed now to actual positions